packages/documentation-spa/blog/2021-08-09-state-of-one-platform-jul-2021.md

```diff
@@ -0,0 +1,97 @@
+---
+slug: state-of-one-platform-jul-2021
+title: State of One Platform - Jul 2021
+author: Ghanshyam Lohar
+author_title: Senior Software Engineer @ Red Hat
+author_url: https://github.com/ghanlohar
+author_image_url: https://avatars.githubusercontent.com/u/5575651?s=400&v=4
+tags: [redhat, blog, oneportal, oneplatform, react, angular, vue, jul, update]
+---
+### Meta
+**Completed Sprints:** OP – Jul'21 Mid Sprint, OP – Jul'21 End Sprint
+
+**Ongoing/Upcoming Sprint:** OP – Aug'21 Mid Sprint
+
+**One Platform:** [one.redhat.com](https://one.redhat.com)
+
+**JIRA:** [https://projects.engineering.redhat.com/projects/ONEPLAT/issues](https://projects.engineering.redhat.com/projects/ONEPLAT/issues)
```

Comment:
```suggestion
**JIRA:** [https://issues.redhat.com/projects/ONEPLAT/issues](https://issues.redhat.com/projects/ONEPLAT/issues)
```

Reply:
@riginoommen We don't need these links at all since even if issues.redhat.com is now external the JIRAs themselves are not accessible from outside